### DISCUSSION TITLE
pilots to run in ARM machine

### DISCUSSION THREAD

**Discussion by zhangxiaomei** [DIRACGrid/DIRAC#6206] [Q&A] [answered]
We were asked to use some ARM resources.  Before trying, I would like to ask if it is possible for pilots to run in ARM machines. How much efforts are needed to make it working? Thanks

**Comment by fstagni** [comment, marked as answer]
Python2 pilots won't work, but Python3 pilots should be already working with ARM (at least DIRACOS2 can be installed). Having said that, I don't think we have ever tried, so it would be beneficial if you try and send us results.

**Reply by zhangxiaomei** [reply]
Thanks. I will have a try and let you know.

**Reply by zhangxiaomei** [reply]
I have a try on ARM with DIRAC7.3.  The tests show the python3 pilots are able to run in ARM machine and pull jobs successfully.  Only a  minor warning with DB12:
"2022-07-07 01:31:37 UTC ExternalLibrary/root WARN: You are executing DB12 using python3, DB12 score is generally higher than it was with python2
2022-07-07 01:31:37 UTC ExternalLibrary/root WARN: Trying to apply a correction...
2022-07-07 01:31:37 UTC ExternalLibrary/root WARN: Cannot correct the score, return the raw norm: cannot access CPU information"
